apps/nrm/translations/src/en/validation.json

```diff
@@ -90,8 +90,8 @@
   },
   "pv-dob": {
     "date": "You must enter a valid date",
-    "before": "Date cannot be in the future",
-    "after": "This date appears to be too far in the past"
+    "before": "You must enter a valid date. The date cannot be in the future",
+    "after": "You must enter a valid date. The date cannot be too far in the past"
```

Comment:
obviously not a coding issue or you @cmedlandho but clearly an inconsistency in the content.  One area says `You must` and other areas says Enter and removes `You must`, one for a content designer whenever we get one